privacy.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: privacy
+title: Privacy Notice
+---
+
+# Foreman Privacy Policy
+
+This document describes the data Foreman collects about you, why it is
+collected, what is done with it, and your options for how it's processed.
+
+The Foreman project does not share data with 3rd parties, except for hosting
+requirements (e.g. GitHub, YouTube) as detailed below, or when required to by
+law.
+
+# Types of data we collect
+
+* Contact details
+  * Name
+  * Email address
+* Data that identifies you
+  * IP addresses
+  * Browser UserAgent strings
+  * Debug archives
+  * Answers to survey questions
+
+# How and why we use your data
+
+## The website
+
+The Foreman website is a static site which does not track, collect, store
+information or set cookies. There is no account for the website itself. The
+webserver (Apache) logs will contain IP addresses, timestamps, and UserAgent
+strings - these are log-rotated and deleted after 4 weeks.
+
+The website also uses Google Analytics, but we only collect anonymous data. IP
+addresses are [anonymised][GA-anonIP], and we do not use User-ID, or any linked
+Google product (e.g. AdWords). Google Analytics data is deleted after 26 months
+(the default for Google Analytics).
+
+The webserver logs and Google Analytics are used to produce aggregated
+statistics about the community, such as popularity of different browsers,
+operating system packages, plugins, etc.
+
+* Legal basis for this data usage: Legitimate interest
+
+### Website videos
+
+The website does embed videos from YouTube, Fosdem, and Slideshare - see the
+links below for details:
+
+  * [YouTube's privacy policy][YouTube]
+  * [FOSDEM homepage](https://fosdem.org)
+  * [SlideShare (LinkedIn) privacy policy][SlideShare]
+
+for details. These videos do not autoplay.
+
+### Website downloads
+
+The downloads from our website are cached by Fastly, a CDN provider. This may
```

Comment:
s/cached/served/ perhaps? Might also be good to explicitly mention `downloads.theforeman.org` and `staging-deb.tf.o` since those are currently served by the CDN, not the main website.

Reply:
I was trying to avoid having to update this policy once we expand the CDN coverage. I can change it if we think more specificity is needed.